sendgrid.js

```diff
@@ -0,0 +1,29 @@
+const sgMail = require("@sendgrid/mail");
+require("dotenv").config();
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+const emailConfig = (email, verificationToken) => {
+  return {
+    to: `${email}`,
+    from: "radekie@wp.pl",
+    subject: "SG Verification Email",
+    text: `To verify your account please click the following link: http://localhost:3000/api/users/verify/${verificationToken}`,
```

Comment:
tę ścieżkę do serwera dałbym do .env, dla produkcji będzie to lepsze rozwiązanie :)